client/components/plans/plan-actions/index.jsx

```diff
@@ -155,12 +162,26 @@ module.exports = React.createClass( {
 			</div>;
 		}
 
+		var classes = classNames( {
```

Comment:
`classNames` is useful when a class depends on some condition, but if these are all always present at this point, we can just use a string in the JSX:

```js
<button className="button is-primary plan-actions__upgrade-button expiry-space-trials-enabled">
```